src/errors.rs

```diff
@@ -42,7 +42,7 @@ impl fmt::Display for MissingImplementationContent {
 }
 
 #[derive(Error, Debug)]
-pub enum OpenLimitError {
+pub enum OpenLimitsError {
```

Comment:
A shame there is a minor API break here.  We will have to make sure to communicate this change to Margin/Velvet.